src/encryption/encryption.go

```diff
@@ -0,0 +1,126 @@
+package encryption
+
+import (
+	"fmt"
+	"keboola-as-code/src/state"
+	"keboola-as-code/src/utils"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/iancoleman/orderedmap"
+	"go.uber.org/zap"
+)
+
+func isKeyToEncrypt(key string) bool {
+	return strings.HasPrefix(key, "#")
+}
+
+func isValueEncrypted(value string) bool {
```

Comment:
1. pouzival by som kratsie nazvy, `isEncrypted` IMHO bohato staci
2. dolezitejsie je, ze uzivatle moze mat v configu nejaku hodnotu, ktoru zasifroval sam, a bude to napr. `KBC::ComponentSecure` ... a tymto by sa mu to rozbilo -> znova zasifrovalo ... teda narvhujem upravit `regexp` na `^KBC::.+::.+$`